Applications/FileManager/main.cpp

```diff
@@ -84,6 +88,11 @@ int main(int argc, char** argv)
         directory_view->open(path);
     };
 
+    directory_view->on_selection = [&](auto& path) {
+        dbg() << "Selected " << path;
+        selected_paths = { FileSystemPath(path) };
```

Comment:
This seems like it would only ever copy/paste one file. What about multiple selection?
Instead, why don't we have `DirectoryView` publish the `on_selection_change` hook from its views.
Then we can iterate over all of the selected paths via `view.selection()` and support copying an arbitrary number of items.